template-resourcegroup.yaml

```diff
@@ -49,7 +49,7 @@ Resources:
     #   - <HOSTNAME2>
 
     # Services is one or more services provided by this resource;
-    # valid services are listed in topology/services.yaml
+    # valid services are listed in topology/services.yaml with the format <SERVICE NAME>: <ID>
```

Comment:
I don't the user needs to know this.

Reply:
We need to clarify so that users aren't entering `CE: 1:` for their service